ext/winevt/winevt_utils.cpp

```diff
@@ -497,7 +497,7 @@ get_description(EVT_HANDLE handle)
 }
 
 VALUE
-render_system_event(EVT_HANDLE hEvent)
+render_system_event(EVT_HANDLE hEvent, BOOL does_preserve_qualifiers)
```

Comment:
`does_` seems unnecessary.

Reply:
Fixed in https://github.com/fluent-plugins-nursery/winevt_c/pull/25/commits/94a22c2a56e9cd2cdff741cc8a13594cbd89f0fd